src/components/admin/PortalAddEditForm.tsx

```diff
@@ -170,13 +170,13 @@ export const PortalAddEditForm: React.FC<PortalAddEditFormProps> = ({
 
   const [
     { loading: isDeleting, error: deleteError },
-    deletePortal,
+    removePortal,
   ] = useAsyncFn(async () => {
     if (!currentSpaceId || !portal) return;
 
-    await deleteRoom(currentSpaceId, portal);
+    await deletePortal(currentSpaceId, portal);
     await onDone();
-  });
+  }, [currentSpaceId, onDone, portal]);
```

Comment:
the fix is here

Reply:
@sunny-viktoryia The same code exists in `ScreeningRoomVideoAddEditForm.tsx` Not sure if the code is still relevant, it's definitely OOS but we might have to fix that one as well.

Reply:
great catch! I'll check.

Reply:
done: https://github.com/sparkletown/sparkle/pull/3000/commits/5ebf45d88e857b548d00b79459c89244c3d6e92e